packages/server/src/service/util.ts

```diff
@@ -50,12 +50,60 @@ export const safeReadFileSync = (
   options?: { encoding?: null | undefined, flag?: string | undefined },
 ) => copyBuffer(fs.readFileSync(path, options))
 
+
+/**
+ * Returns a string indicating the current week number. Since this function
+ * follows ISO-8601, the first week of January might actually belong to
+ * the previous year, which is why this function returns a string formatted
+ * as `YYYY.WN` where YYYY is the year and WN the weeknumber.
+ *
+ * @param d exported for testing purposes, defaults to now when not present
+ *
+ * Based on: https://stackoverflow.com/a/6117889/9611230
+ */
+export const weekNumber = (d: Date = new Date()): string => {
+  const date = new Date(Date.UTC(d.getFullYear(), d.getMonth(), d.getDate()))
+
+  // Set to nearest Thursday: current date + 4 - current day number
+  // Make Sunday's day number 7
+  date.setUTCDate(date.getUTCDate() + 4 - (date.getUTCDay()||7))
+
+  const firstDayOfYear = new Date(Date.UTC(date.getUTCFullYear(), 0, 1))
+  // Calculate full weeks to nearest Thursday
+  const number = Math.ceil(
+    (((date.valueOf() - firstDayOfYear.valueOf()) / 86400000) + 1) / 7
+  )
+
+  // If the month is January and week number is above 5, it belongs to the
+  // previous year
+  const previousYear = date.getUTCMonth() === 0 && number > 5
+  const year = previousYear ? date.getUTCFullYear() - 1 : date.getUTCFullYear()
+
+  return `${year}.${number}`
+}
+
+// Returns a boolean indicating if the cache should be updated, also updates
+// the cacheVer file when needed.
+const handleCacheVer = (options?: Options): boolean => {
+  const yearWeek = weekNumber()
+  const workDir = options?.workDir ?? `${__dirname}/cache`
+  const path = `${workDir}/cacheVer`
+  const encoding = 'utf-8'
+
+  if (fs.existsSync(path)) {
```

Comment:
This seems pretty inconsistent -- what is the file name suppose to be?  It seems like you are using an inconsistent filename